test/helper_non-windows_test.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
```

Comment:
bsd/darwin is not unix - so I suggest the other naming should be changed, not this. !windows _means_ non-windows - they are equivalent. unix does not mean non-windows - those are not the only two variants.